README.md

```diff
@@ -50,16 +79,22 @@ Usage of ./chinadns:
   -reuse-port
         Enable SO_REUSEPORT to gain some performance optimization. Need Linux>=3.9 (default true)
   -s value
-        Upstream DNS servers. Need China route list to check whether it's a trusted server or not. (default 119.29.29.29,114.114.114.114)
+        Comma separated list of upstream DNS servers. Need China route list to check whether it's a trusted server or not.
+        Servers can be in format ip:port or protocol[+protocol]@ip:port where protocol is udp or tcp.
+        Protocols are used in the order they are defined (left to right).
+        If empty, protocol defaults to udp+tcp and port defaults to 53.
+        Examples: udp@8.8.8.8,udp+tcp@127.0.0.1:5353,1.1.1.1 (default udp+tcp@119.29.29.29,udp+tcp@114.114.114.114)
   -test-domains string
         Domain names to test DNS connection health. (default "qq.com,163.com")
   -timeout duration
         DNS request timeout (default 1s)
   -trusted-servers value
-        Servers which (located in China but) can be trusted. (default 193.112.15.186:2323)
+        Comma separated list of  servers which (located in China but) can be trusted.
+        Uses the same format as -s.
   -udp-max-bytes int
-        Default DNS max message size on UDP. (default 1410)
+        Default DNS max message size on UDP. (default 4096)
```

Comment:
Thanks for the fix 😄

Reply:
🤣 I hadn't noticed this one, I just copy pasted everything